### ISSUE TITLE
IdaSkin not start at First QuickPage Load/But old 1.4 version work well 

### ISSUE THREAD

**Issue by vanasisf** [zyantific/IDASkins#45]
I tested the old version of IDASkins.
In the old version plugin patch start immediately when i start my IDA.
i mean from here
![idapro1](https://user-images.githubusercontent.com/10735010/45584966-c8c84f80-b8f1-11e8-8d91-fcf43d76c5f6.png)
and it works well even at this page.
but in new version ( last build ) the IDASkin plugin start after main ida page load completely. after that the IDASkin works normally. but if i close it, again i need to go main ida page to see dark theme

steps in old version ( 1.4 )
1- Click at IDA icon in desktop
2- now see IDA QUICKSTART page
3- then i see idaskin plugin firstuse menu
4- select dark or any other themes
5- close app
6- open again, now i see dark theme for QUICKSTART page too.

steps in new version ( 2.x)
1- Click at IDA icon in desktop
2- now see IDA QUICKSTART page
3- here i cant see idaskin plugin firstuse menu ( i delete the regedit files too )
4- i need to click on new/Go/preview buttons to open IDA main page
5- when opened ida main page, i see the idaskin plugin firstuse menu
6- now select dark or any theme.
7- close app
8- now i see QUICKSTART page without any theme ( it is white )

I guess in new version the plugin starts after ida full load, not in Quickstart page
note1: if i load the main page, and after that open the Quickstart page i can see dark theme.
but if i open program directly from desktop icon, i need to go main page with clicking some button in quickstart page to load ida . after that i can see the dark theme.

App info:
IDA version : V7.0.170914
Old plugin version that i used : 1.4
new version that i used : 2.x ( last version )

**Comment by athre0z** [comment]
v2.0 was rewritten in Python and it appears as if these are loaded only after the main windows is opened. I don't think there is much we can do about that, sorry.

**Comment by vanasisf** [comment]
this is what i mean for ida **main page**
![idapro2](https://user-images.githubusercontent.com/10735010/45585089-e1396980-b8f3-11e8-9a3e-9139a8131d64.png)

and this is the **QuickStart page**
![idapro3](https://user-images.githubusercontent.com/10735010/45585094-ec8c9500-b8f3-11e8-89aa-1d2209bd4783.png)

and this is the **IDA ICON**
![idapro4](https://user-images.githubusercontent.com/10735010/45585101-0d54ea80-b8f4-11e8-864e-5201a27bb7f8.png)


So in old version when i click the **IDA ICON** i see a dark theme for Quick start page like 
![idapro3](https://user-images.githubusercontent.com/10735010/45585110-34abb780-b8f4-11e8-8a72-a69293b3616b.png)

but in new version of idaskin i see a white theme for Quickstart page
![idapro1](https://user-images.githubusercontent.com/10735010/45585117-55740d00-b8f4-11e8-80fc-90b9e17f4188.png)






Important note : in both versions after **main page** load , i see a dark theme for QuickStart page
![idapro2](https://user-images.githubusercontent.com/10735010/45585147-fbc01280-b8f4-11e8-9e6b-3e4306a54543.png)

**Comment by vanasisf** [comment]
is it possible to convert python files to dll like old version? ( with compiling )?

**Comment by athre0z** [comment]
Possibly, but I don't think it's worth the effort. I rewrote it in Python in order to ditch the cancer of performing native builds for all platforms and this would be a step back towards platform specific stuff. Also, it wouldn't fix the issue on macOS and Linux (without equivalent fixes for these platforms).

**Comment by vanasisf** [comment]
thanks for fast answer

**Comment by athre0z** [comment]
Closing as `wontfix` for now.
